src/app/Caches/CacheMetrics.tsx

```diff
@@ -218,17 +219,17 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
             <TextList component={TextListVariants.dl}>
               <TextListItem component={TextListItemVariants.dt}>
                 {displayUtils.formatNumber(
-                  stats.current_number_of_entries_in_memory
+                  stats.off_heap_memory_used
                 )}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Current number
+                Heap Memory
               </TextListItem>
               <TextListItem component={TextListItemVariants.dt}>
                 {displayUtils.formatNumber(stats.data_memory_used)}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Total in use
+                Data Memory
```

Comment:
"Amount of memory in use" How is this different to "Amount of heap"? Does this show heap + off-heap?

Reply:
> "Amount of memory in use" How is this different to "Amount of heap"? Does this show heap + off-heap?

@karesti ?

Reply:
@danberindei could you please add some light on these metrics meaning please? :)

Reply:
`data_memory_used` is the amount of heap used to store cache entries, IF the cache is using heap storage, eviction is enabled, and size-based (`<memory storage="HEAP" max-size="some pozitive number"/>`).

`off_heap_memory_used` is the amount of off-heap/native memory used, if the cache is using off-heap storage, regardless of whether eviction is enabled or not (`<memory storage="OFF_HEAP"/>`).

So the labels should probably be "Heap memory" for `data_memory_used` and "Off-heap memory" for `off_heap_memory_used`.

If the console knows whether the cache is using off-heap storage or not, then it should only show the relevant memory stat. If not, then it should display a "N/A" when the stat is missing/-1, and the label (or maybe a tooltip?) could say something like
* `data_memory_used`: Only available when cache is configured to use heap storage and size-based eviction is enabled".
* `off_heap_memory_used`: Only available when cache is configured to use off-heap storage.

Speaking of tooltips, the `data_memory_used` stat only includes memory used to store cache entries, not memory used by internal data structures (e.g. to keep track of transactions, or to keep track of modified keys during state transfer). We might want to have a tooltip even when the stat is present, saying something like "This number only tracks cache entries, total memory usage is higher because of internal data structures that are not tracked".

Reply:
Thanks @danberindei 

@dipguptaredhat and @karesti What do you think about adding tooltips/hover help here as Dan suggests? I think for both stats we could probably have a single bit of hover help like:

"Displays the total amount of memory that data in the cache uses. The statistic that is available reflects cache configuration, which can be either JVM heap or off-heap, native memory. Total memory usage is higher because it includes internal data structures."

Reply:
yes we are adding those tooltips